sam-react-app/coreconcept/src/movieForm/MovieForm.tsx

```diff
@@ -0,0 +1,138 @@
+import React, {ChangeEvent, Component, KeyboardEvent} from "react";
```

Comment:
Add styling to the forms. It doesn't have to be pixel perfect. No need to render components from previous chapters.